packages/teams-js/src/private/externalAppAuthentication.ts

```diff
@@ -133,6 +135,18 @@ export namespace externalAppAuthentication {
     data: string | Record<string, unknown>;
   }
 
+  /**
+   * @hidden
+   * @internal
+   * Limited to Microsoft-internal use
+   */
+  export class SerializableActionExecuteInvokeRequest implements ISerializable {
+    public constructor(private invokeRequest: externalAppAuthentication.IActionExecuteInvokeRequest) {}
+    public serialize(): object | string {
+      return this.invokeRequest; // I wonder if the Record actually needs to be serialized. Might be a real bug?
```

Comment:
Record is consisting of only primitive types that are serialized internally in post message. This means we can pass the data without serializing them. I will put a PR out that will make sure the Record actually contains only primitive types. 

Automatic serialization would not happen on `symbol` type that is a primitive type. I am talking to 3pcxp to make sure that won't be sent as part of Record.